src/layouts/components/Editor/extensions/Details/IsomerDetails.ts

```diff
@@ -68,4 +68,27 @@ export const IsomerDetails = Details.extend({
       )
     }
   },
+
+  addKeyboardShortcuts() {
+    return {
+      Backspace: () => {
+        const { schema, selection } = this.editor.state
+        const { empty, $anchor } = selection
+
+        if (!empty || $anchor.parent.type !== schema.nodes.detailsSummary) {
+          return false
+        }
+
+        if ($anchor.parentOffset !== 0) {
+          return this.editor.commands.command(({ tr }) => {
+            const start = $anchor.pos - 1
+            const end = $anchor.pos
+            tr.delete(start, end)
+            return true
+          })
+        }
+        return this.editor.commands.removeDetail()
+      },
+    }
```

Comment:
idt this works - adding a keyboard shortcut would invoke it for every associated keypress (`Backspace`) here (unless we copied over the backspace logic from tiptap)  

i think if the intended outcome is to delete the block as a whole, we might wanna just make the parent node `defining` so that backspace consumes it atomically.

Reply:
oh the purpose of this is not to delete the block as a whole, is to actually have a functional backspace key press for the titles, but I should not be able to backspace on an empty title to delete the entire block (deletion of the entire block should be in the button)!

Reply:
that's fine, but because this triggers for every keypress on backspace, it might destructively interfere with other nodes that don't adhere to the default backspace behaviour.

have we tried deleting:
1. nested nodes (cards) with this?
2. `defining` nodes?
3. plain text?

as this uses the `selection` and checks if the selection is empty, we might mess up on deletion on plain text w/o a selection (haven't tried)

Reply:
hey good point!
regarding the selection, the delete button selects the node as a whole, and therefore this should not accidently delete another selected text for example. 

regarding the 
- nested card, deletion works as expected, the cards is deleted as expected 
- defining nodes, eg text? abit confused about this point ps, can clarify here 
- plain text works too

Reply:
`defining` nodes are those that should be deleted as a group! (i think iframe is, for one)

if this works, then am ok w/ it as chance of destructive interference seems low - tiptap might dispatch the event to the specific node itself, in which case idt my concerns stand.